src/_pytest/debugging.py

```diff
@@ -293,7 +294,9 @@ def pytest_exception_interact(
             sys.stdout.write(out)
             sys.stdout.write(err)
         assert call.excinfo is not None
-        _enter_pdb(node, call.excinfo, report)
+
+        if not isinstance(call.excinfo.value, unittest.SkipTest):
+            _enter_pdb(node, call.excinfo, report)
```

Comment:
Can you add a test that we still *do* enter `pdb` if `unittest.SkipTest` is raised within a test function?

Reply:
Do you think we should enter `pdb` in that case?  I would argue we shouldn't.

Reply:
And thanks for promptly reviewing this PR :)

Reply:
From the [python docs](https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures):

> Skipping a test is simply a matter of using the [skip()](https://docs.python.org/3/library/unittest.html#unittest.skip) [decorator](https://docs.python.org/3/glossary.html#term-decorator) or one of its conditional variants, calling [TestCase.skipTest()](https://docs.python.org/3/library/unittest.html#unittest.TestCase.skipTest) within a [setUp()](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp) or test method, or raising [SkipTest](https://docs.python.org/3/library/unittest.html#unittest.SkipTest) directly.

If those three actions are functionally equivalent, and we don't break into `pdb` for the `skip()` decorator, then it seems like we shouldn't break into `pdb` for raising `SkipTest`.

Reply:
Oops, yep, skipping both is consistent with the behavior of `pytest.skip()` too.  Merging 🚀